backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Models/AuthenticationResponse.java

```diff
@@ -0,0 +1,11 @@
+package deti.uas.uasmartsignage.Models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class AuthenticationResponse {
+    
+    private final String jwt;
+}
```

Comment:
having the role and email daria jeito

Reply:
There's an endpoint (/lapi/login/info) that returns that info. Already done in frontend.

Reply:
entao tenho de fazer login e logo em seguida tenho de chamar outro endpoint para obter essa info miseria